docker_deep_dive_part_1/Deploy_Wordpress_With_Multiple_Container.md

```diff
@@ -0,0 +1,66 @@
+## Deploy Wordpress With Multiple Container 
+In this section we deploy WordPress Website With Multiple Container
+
+Start off by making a new directory where you wish to store the files for WordPress and MariaDB for example in your home directory.
+```
+mkdir ~/wordpress && cd ~/wordpress
+```
+
+### Docker Setup For Database Server 
+
+Search Mariadb Image
+```
+docker search mariadb
```

Comment:
Do we required explicit search. Can't direct pull work ?

Reply:
i use that command is just to people aware about that command also.(before pull they can check that which images able related to that search) .